spec/controllers/openid_application_controller_spec.rb

```diff
@@ -221,6 +221,18 @@ def index
       end
     end
 
+    context 'with an opaque token supplied and no session' do
+      it 'returns 200 and add the user to the session' do
+        with_okta_configured do
```

Comment:
Is Okta configuration necessary here since we don't actually get that far?

Reply:
Not necessary since this is for the non-Okta case.

I have removed the `with_okta_configured` wrapper and confirmed the test still succeeds.